src/Device/DeviceSelector/DeviceSelector.tsx

```diff
@@ -93,6 +94,13 @@ const DeviceSelector: FC<Props> = ({
 
     useAutoselectDevice(doSelectDevice);
 
+    useHotKey({
+        hotKey: 'alt+s',
+        title: 'Select device',
+        isGlobal: true,
+        action: () => toggleDeviceListVisible(),
+    });
```

Comment:
We were actually working on this before we decided to finish the PR, but decided together with @jonasem that it could be a later feature, as we have already spent a lot of time on this project and have another (large one) to finish before summer ends.

But we agree, this will be the next feature if we finish the next project early.